### ISSUE TITLE
when to use logit layer

### ISSUE THREAD

**Issue by WatershedXiaolan** [WatershedXiaolan/Xiaolan-s-Gitbook#170]
usually before softmax layer
<img width="601" alt="Screen Shot 2022-02-25 at 4 26 34 PM" src="https://user-images.githubusercontent.com/5552371/155811480-68bc4e69-d17f-4ea4-b295-5a664eaf293f.png">
